tests/ui/const-generics/late-bound-vars/in_closure.stderr

```diff
@@ -0,0 +1,13 @@
+error: internal compiler error: compiler/rustc_borrowck/src/universal_regions.rs:882:36: cannot convert `ReFree(..., BrNamed(..., 'a))` to a region vidquery stack during panic:
```

Comment:
We need to normalize the source path, to avoid meaningless changes.

Reply:
doneish